src/snake_game/handlers.cljs

```diff
@@ -1,86 +0,0 @@
-(ns snake-game.handlers
-  (:require-macros [reagent.ratom :refer [reaction]])
-  (:require [re-frame.core :refer [register-handler
-                                   register-sub
-                                   dispatch]]
-            [goog.events :as events]
-            [snake-game.utils :as utils]))
-
-;; Define app data. We need to define our board, points and snake. In our snake vector is in map desctibed position :position of every 'snake body' part. First elemeny in vector id head of snake. Key :direction is the direction of the next move.
-
-(def board [35 25])
-
-(def snake {:direction [1 0]
-            :body [[3 2] [2 2] [1 2] [0 2]]})
-
-(def initial-state {:board board
-                    :snake snake
-                    :point (utils/rand-free-position snake board)
-                    :points 0
-                    :game true})
-
-(register-handler                  ;; setup initial state
- :initialize                       ;; usage (submit [:initialize])
- (fn
-   [db _]
-   (merge db initial-state)))      ;; what it returns becomes the new @db state
-
-(register-handler
- :next-state
- (fn
-   [db _]
-   (if (:game db)
-     (if (utils/collisions db)
-       (assoc-in db [:game] false)
-       (-> db
-           (update-in [:snake] utils/move-snake)
-           (as-> after-move
```

Comment:
This does not reads very clean to me, especially because it is in the other `->` you can probably just pass `db` to process-move unless I am misreading something.